ddp-workspace/projects/ddp-pancan/src/app/components/participation/participation.component.scss

```diff
@@ -0,0 +1,83 @@
+@import '../../../styles/common.scss';
+
+.participation {
+    text-align: left;
+
+    .title {
+        @extend .static-page-title;
+        margin: 4.56rem auto 4.81rem auto;
+        max-width: 1440px;
+        text-align: center;
+    }
+
+    .step {
+        padding: 4.94rem 1.5rem 5rem 1.5rem;
+        display: flex;
+        flex-direction: row-reverse;
+        justify-content: space-around;
+        gap: 8rem;
+
+        &:nth-of-type(odd) {
+            background-color: $section-grey-background-color;
+            padding-bottom: 3rem;
+            flex-direction: row;
+        }
+
+        &__data {
+            max-width: 700px;
+        }
+
+        &__label {
+            @extend .participate-step-label;
```

Comment:
#minor what do you think about attaching the class to the html element instead of using extend? after conversion scss to css we get the next code
.participate-step-label,
.participation step__label {
 ... // styles for .participate-step-label
}
 which a little bit harder to debug when we want to disable some styles only for one element. On the other hand, assigning the simple class to the html element allows us to enable/disable this class for the certain element in the devtools for example.

Reply:
I'd recommend using extend in case when just assigning the class doesn't work because of css specificity

Reply:
I used `@extend` as to re-use common styles for different elements there.
I believe the best choice would be using `scss mixin` instead of `extend`

Reply:
Finally reverted. (see https://github.com/broadinstitute/ddp-angular/pull/661#discussion_r657472528)